test/resources/apiKey.js

```diff
@@ -19,10 +19,6 @@ describe('ApiKey Resource', () => {
     expect(apiKey).to.be.a('function');
   });
 
-  it('throws on retrieve', () => ApiKey.retrieve().then(() => {}, err => {
-    expect(err).to.be.an.instanceOf(NotImplementedError);
-  }));
-
```

Comment:
Can we add a real test?

Reply:
Isn't this tested via `base.js`? shipments, addresses, etc do not have an explicit retrieve test in their test files.

Reply:
Ya, it is.